src/main/java/com/autotune/analyzer/workerimpl/BulkJobManager.java

```diff
@@ -128,42 +130,35 @@ public void run() {
                         createExecutor.submit(() -> {
                             String experiment_name = apiObject.getExperimentName();
                             BulkJobStatus.Experiments newExperiments = jobData.getData().getExperiments();
+                            BulkJobStatus.Experiments failedExperiments = jobData.getData().getExperiments();
                             BulkJobStatus.RecommendationData recommendationData = jobData.getData().getRecommendations().getData();
                             try {
-                                ValidationOutputData output = new ExperimentDBService().addExperimentToDB(apiObject);
-                                if (output.isSuccess()) {
+                                // skip appending the experiments of type - job for now until the 'job' workload is fully supported
+                                if (experiment_name.contains(AnalyzerConstants.K8sObjectConstants.Types.JOB)) {
```

Comment:
This conditions should be removed as failed already handled above during createExperiment APU